pages/common/aws-pricing.md

```diff
@@ -0,0 +1,24 @@
+# aws pricing
+
+> Centralized and convenient way to programmatically query Amazon Web Services for services, products, and pricing information.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/pricing/>.
+
+- Get a list of service codes to query later for:
+
+`aws pricing describe-services --region {{us-east-1}}`
+
+- Get a a list of attributes for a given service code:
+
+`aws pricing describe-services --service-code {{AmazonEC2}} --region {{us-east-1}}`
+
+- Get pricing information for a service code:
+
+`aws pricing get-products --service-code {{AmazonEC2}} --region {{us-east-1}}`
+
+- Get a list of attribute values to use while filtering:
+
+`aws pricing get-attribute-values --service-code {{AmazonEC2}} --attribute-name {{instanceType}} --region {{us-east-1}}`
+
+- Get pricing information for a service code and attribute using a filter on the instance type and location:
+
+`aws pricing get-products --service-code {{AmazonEC2}} --filters {{"Type=TERM_MATCH,Field=instanceType,Value=m5.xlarge"}} {{"Type=TERM_MATCH,Field=location,Value=US East (N. Virginia)"}} --region {{us-east-1}}`
```

Comment:
Our standard is quotes outside the token syntax
```suggestion
`aws pricing get-products --service-code {{AmazonEC2}} --filters "{{Type=TERM_MATCH,Field=instanceType,Value=m5.xlarge}}" "{{Type=TERM_MATCH,Field=location,Value=US East (N. Virginia)}}" --region {{us-east-1}}`
```

Reply:
Sorry for that! Will consider it for future contributions...